packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js

```diff
@@ -81,6 +83,22 @@ class DateTimeView extends React.PureComponent {
 		);
 	}
 
+	renderValidationButton(t) {
+		const validateLabel = t('DATEPICKER_VALIDATE_BUTTON', { defaultValue: 'Done' });
+		return (
+			<div className={theme.footer}>
+				<Action
+					name="action-datepicker-validate"
+					bsStyle="primary"
+					role="button"
+					label={validateLabel}
+					onClick={this.props.onFinish}
+					disabled={!this.props.isValid}
```

Comment:
You don't have any indication where the error is. Hard for user to fix. So IMO you should not disable it.
Let the error system do its job. (validate that with ux, exposing this point of vue)
If so, you don't need the isValid props anymore